src/components/common/table.module.scss

```diff
@@ -12,58 +17,65 @@
     white-space: nowrap;
   }
 
-  caption,
-  th {
+  caption {
     text-align: left;
+    font-weight: bold;
+    font-size: 1.05rem;
+    margin-bottom: 1rem;
   }
 
-  td,
-  th[scope='col'] {
-    text-align: right;
-  }
-
-  caption {
-    font-weight: bold;
-    font-size: 1.25rem;
-    padding: 0.75rem 0;
+  colgroup:not(:first-child) {
+    border-left: 1px solid $color-slate-300;
   }
 
-  th,
   td {
-    border: 1px solid #c0c0c0;
-    padding: 0.25rem;
-  }
+    text-align: right;
+    padding: 0.75rem 0.5rem;
+    font-size: 1rem;
 
-  th:last-child {
-    padding-right: 0.25rem;
+    &.align-left {
+      text-align: left;
+    }
   }
 
-  th {
-    background: #dee2e4;
+  thead {
+    border-bottom: 1px solid $color-slate-400;
 
-    span {
-      display: block;
-      font-size: 0.75rem;
-    }
+    th {
+      padding: 0 0.5rem 0.5rem;
+      vertical-align: bottom;
 
-    &[scope='colgroup'] {
-      font-size: 1.1em;
-      text-align: center;
-    }
-  }
+      &[scope='col'] {
+        text-align: right;
+        font-size: 0.8rem;
+        font-weight: 800;
+      }
```

Comment:
Just curious: could these two selectors be simplified to just the `th`? (I’d assumed all the `thead th` elements would be `scope="col"`.)

(Also, not a blocker for merge; really just curious!)

Reply:
Good call! The additional level of nesting isn't needed, will fix.